gitpod-io/website
# fix SSO availability in pricing table

## Description

Change SSO availability from "coming soon" to available for Dedicated in the pricing page's feature table

## How to test

Go to the preview, navigate to pricing page, check that there is a green check mark for SSO in the dedicated column of the feature table.
